### DISCUSSION TITLE
What kind of git token should I create?

### DISCUSSION THREAD

**Discussion by lsunsi** [release-plz/release-plz#1106] [Q&A] [answered]
I just made my first release and got a `git release not configured. Did you specify git-token and backend?`.
Fine, all good, makes sense.

Should I create a fine-grained token? Which permissions does it need?
Or maybe a classic token?

Sorry if this is written somewhere, I looked on the documentation and didn't find anything.
Thanks!

**Comment by marcoieni** [comment, marked as answer]
Did you miss this [page](https://release-plz.ieni.dev/docs/github)?
If yes, please tell me why so I can improve the website :)

**Reply by lsunsi** [reply]
I did miss it, but it doesn't answer my question. Let me try to be clearer: I'm running the CLI `release-plz release` (and loving it), so I still have the question of what to generate manually on GitHub so the command works!

**Reply by marcoieni** [reply]
Ooops, sorry.
The doc you are looking for is [here](https://release-plz.ieni.dev/docs/github/token). I should probably change the structure of the docs 👍

**Reply by lsunsi** [reply]
Worked perfectly! Yeah, it was a little hard to find, but you've done an amazing job with this library. Thanks!
Also, thanks for the quick responses

**Reply by wmmc88** [reply]
> Ooops, sorry. The doc you are looking for is [here](https://release-plz.ieni.dev/docs/github/trigger#use-a-personal-access-token). I should probably change the structure of the docs 👍

If anyone in the future reads this, the linked page has moved to https://release-plz.ieni.dev/docs/github/token

**Reply by marcoieni** [reply]
edited my message, thanks!
